### PR TITLE
Nc games.2

### PR DESCRIPTION
**Pull request by robbiemcdonough**

### CODE REVIEW

**Review by Harrison365 — CHANGES_REQUESTED** [review]
I cant see any read me changes 😢

**Review by hevdevs — CHANGES_REQUESTED** [review]
Hi Robbie - we just want to be telling the person viewing your portfolio project how to setup the project. Given that the bulk of the setup is done for them with the connection.js already, all you need to tell them to do is to first run npm install to install all of the dependencies, then create a .env.test and .env.development file with the PGDATABASE=nc_news/nc_news_test. Hope that makes sense! Put in another PR once this is done

File: README.md [hunk 1]
```diff
@@ -6,52 +6,11 @@ We will be building an API for the purpose of accessing application data program
...
-```
-
-## Husky
-
-To ensure we are not commiting broken code this project makes use of git hooks. Git hooks are scripts triggered during certain events in the git lifecycle. Husky is a popular package which allows us to set up and maintain these scripts. This project makes use a _pre-commit hook_. When we attempt to commit our work, the script defined in the `pre-commit` file will run. If any of our tests fail than the commit will be aborted.
-
-The [Husky documentation](https://typicode.github.io/husky/#/) explains how to configure Husky for your own project as well as creating your own custom hooks.\_
+You will need to create two .env files for your project: .env.test and .env.development. 
+Into .env.test add PGDATABASE=NC_games.test. 
+Into .env.development add PGDATABASE=NC_games. Double check that these .env files are .gitignored.
```
Comment (Liam310) [inline README.md]: Looks good - just be careful about the _exact_ database info. Make sure the casing is right, so all lowercase and have all the words separated by underscores, e.g.`nc_games_test`

File: db/data/development-data/index.js [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+
+const pathToEnvFile = `${__dirname}/.env.${ENV}`;
+
+console.log(pathToEnvFile);
+
+require("dotenv").config({
+  path: pathToEnvFile,
+});
+
+module.exports = new Pool();
```
Comment (Liam310) [inline db/data/development-data/index.js]: This connection logic is already setup for you in the `connection.js` file - no need to add it here! In fact, trying to export a database connection will mess with the behaviour of the rest of your app. Just remove it 🙂

File: db/data/test-data/index.js [hunk 3]
```diff
@@ -2,3 +2,16 @@ exports.categoryData = require('./categories.js');
...
+
+const pathToEnvFile = `${__dirname}/.env.${ENV}`;
+
+console.log(pathToEnvFile); 
+
+require('dotenv').config({
+  path: pathToEnvFile,
+});
+
+module.exports = new Pool();
```
Comment (Liam310) [inline db/data/test-data/index.js]: Like I said for the other file - no need for this logic here!

**Review by Liam310 — COMMENTED** [review]
Looks ready to merge!

**Pull request merged**